.github/workflows/cd.yml

```diff
@@ -0,0 +1,72 @@
+name: merge-build
+on:
+  push:
+    branches:
+      - master
+      - cloud
+      - release-*
+defaults:
+  run:
+    shell: bash
+jobs:
+  check-mattermost-vet:
+    name: Check style
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout mattermost-server
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Checkout focalboard
+        run: |
+          cd ..
+          git clone --depth=1 --no-single-branch https://github.com/mattermost/focalboard.git
+          cd focalboard
+          git checkout $GITHUB_HEAD_REF || git checkout $GITHUB_BASE_REF || git checkout rolling-stable
+          echo $(git rev-parse HEAD)
+          cd ../mattermost-server
+          make setup-go-work
+      - name: Reset config
+        run: make config-reset
+      - name: Run plugin-checker
+        run: make plugin-checker
+      - name: Run mattermost-vet
+        run: make vet BUILD_NUMBER='${GITHUB_HEAD_REF}' MM_NO_ENTERPRISE_LINT=true MM_VET_OPENSPEC_PATH='${PWD}/../mattermost-api-reference/v4/html/static/mattermost-openapi-v4.yaml'
+  test-postgres-binary:
+    name: Run tests on postgres with binary parameters
+    needs: check-mattermost-vet
+    uses: ./.github/workflows/test.yml
+    with:
+      datasource: postgres://mmuser:mostest@postgres:5432/mattermost_test?sslmode=disable&connect_timeout=10&binary_parameters=yes
+      drivername: postgres
+      racemode: -race
+  test-postgres-normal:
+    name: Run tests on postgres
+    needs: check-mattermost-vet
+    uses: ./.github/workflows/test.yml
+    with:
+      datasource: postgres://mmuser:mostest@postgres:5432/mattermost_test?sslmode=disable&connect_timeout=10
+      drivername: postgres
+      racemode: -race
+  test-mysql:
+    name: Run tests on mysql
+    needs: check-mattermost-vet
+    uses: ./.github/workflows/test.yml
+    with:
+      datasource: mmuser:mostest@tcp(mysql:3306)/mattermost_test?charset=utf8mb4,utf8&multiStatements=true
+      drivername: mysql
+      racemode: -race
+  sentry:
+    name: Send build info to sentry
+    runs-on: ubuntu-22.04
+    needs:
+      - test-postgres-binary
+      - test-postgres-normal
+      - test-mysql
+    steps:
+      - name: Checkout mattermost-server
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Install sentry-cli
+        run: curl -sL https://sentry.io/get-cli/ | SENTRY_CLI_VERSION="1.64.1" sh
```

Comment:
Maybe decouple the `SENTRY_CLI_VERSION` from being hardcoded as part of the run, and expose this as an env in the step. 

https://docs.github.com/en/actions/learn-github-actions/variables